docker-compose/docker-compose.multiserver.mariadb.yml

```diff
@@ -13,9 +13,11 @@
 # limitations under the License.
 
 version: "3"
+
 volumes:
   data: {}
   db: {}
+  assets:
```

Comment:
This creates a new volume named `assets`, but it is never used. The `./assets/` volume is local on the filesystem and not directly managed by Docker. This should be removed.

Reply:
Yes, this was a solution I found when the database entry point was not executed.

Then after making more tests and reading the log files, it was de DDL file was not the correct.